### PR TITLE
WIP: Direct paging improvements

### PR DESCRIPTION
**Pull request by iskhakov**
# What this PR does
* Create Direct Paging integration (with default route) when team is created with bulk_update
* Create notification policies when user is created with bulk_update
* If user notification policies are empty change it to Email
* Minor markup and wording improvements
* Add grafana queue to helm chart
* Remove disabled commands for redis helm chart
* Improve Dockerfile caching

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: engine/apps/alerts/models/alert_receive_channel.py [hunk 1]
```diff
@@ -230,10 +230,25 @@ def get_default_template_attribute(self, render_for, attr_name):
...
     @classmethod
     def create(cls, **kwargs):
+        organization = kwargs["organization"]
+        team = kwargs["team"] if kwargs["team"] else None
+        integration = kwargs["integration"]
+        if (
+            integration == AlertReceiveChannel.INTEGRATION_DIRECT_PAGING
+            and AlertReceiveChannel.objects.filter(
+                organization=organization, team=team, integration=integration
+            ).exists()
```
Comment (iskhakov) [inline engine/apps/alerts/models/alert_receive_channel.py:247]: This has been moved to .create() from .save() as we need to check for suplicates only during creation

File: grafana-plugin/src/components/Policy/EscalationPolicy.tsx [hunk 2]
```diff
@@ -73,9 +73,11 @@ export class EscalationPolicy extends React.Component<EscalationPolicyProps, any
...
         backgroundColor={backgroundColor}
       >
-        <WithPermissionControlTooltip disableByPaywall userAction={UserActions.EscalationChainsWrite}>
-          <DragHandle />
-        </WithPermissionControlTooltip>
+        {!isDisabled && (
+          <WithPermissionControlTooltip disableByPaywall userAction={UserActions.EscalationChainsWrite}>
+            <DragHandle />
+          </WithPermissionControlTooltip>
+        )}
```
Comment (iskhakov) [inline grafana-plugin/src/components/Policy/EscalationPolicy.tsx:80]: do not allow dragging disabled objects

File: engine/apps/user_management/models/user.py [hunk 3]
```diff
@@ -75,25 +75,46 @@ def sync_for_team(team, api_members: list[dict]):
...
+                )
+                policies_to_create.append(
+                    UserNotificationPolicy(
+                        user=user,
+                        step=UserNotificationPolicy.Step.NOTIFY,
+                        notify_by=settings.EMAIL_BACKEND_INTERNAL_ID,
+                        order=0,
+                        important=True,
+                    ),
+                )
```
Comment (joeyorlando) [inline engine/apps/user_management/models/user.py:114]: maybe dedup this creation logic (duplicates [this](https://github.com/grafana/oncall/pull/3064/files#diff-6e8b2e38e6eee048bf3817babbd042b94d5752f43b20cf300509e77c1c40aa50)) by creating a method on `User` or `UserNotificationPolicy` model (whichever you feel is more appropriate)

**Review by joeyorlando — APPROVED** [review]
LGTM! One minor comment

**Review by teodosii — APPROVED** [review]

**Pull request merged**